plugins/cache-caffeine/src/test/java/org/opensearch/cache/store/CaffeineHeapCacheTests.java

```diff
@@ -345,51 +316,46 @@ public void testInvalidateConcurrently() throws Exception {
             threads[j].start();
             j++;
         }
-        phaser.arriveAndAwaitAdvance(); // Will trigger parallel puts above.
+        phaser.arriveAndAwaitAdvance(); // Will trigger parallel invalidations above.
         countDownLatch.await();
 
         for (ICacheKey<String> removedKey : removedKeyList) {
             assertNull(caffeineTest.get(removedKey));
         }
         assertEquals(keyValueMap.size() - removedKeyList.size(), caffeineTest.count());
-        caffeineTest.close();
+        assertEquals((keyValueMap.size() - removedKeyList.size()) * 10, caffeineTest.stats().getTotalSizeInBytes());
     }
 
     public void testEvictions() throws Exception {
-        ToLongBiFunction<ICacheKey<String>, String> weigher = getMockWeigher();
+        ToLongBiFunction<ICacheKey<String>, String> weigher = getMockWeigher(false);
         MockRemovalListener<String, String> removalListener = new MockRemovalListener<>();
-        ICache<String, String> caffeineTest = new CaffeineHeapCache.Builder<String, String>()
-            .setDimensionNames(getMockDimensions())
+        ICache<String, String> caffeineTest = new CaffeineHeapCache.Builder<String, String>().setDimensionNames(getMockDimensions())
             .setExecutor(Runnable::run)
             .setExpireAfterAccess(TimeValue.MAX_VALUE)
-            .setMaximumWeightInBytes(100)
+            .setMaximumWeightInBytes(TEST_MAX_CACHE_SIZE)
```

Comment:
This can probably just be a constant variable in the scope of the two evictions tests, called `MAX_CACHE_SIZE` or something. Rn it's a bit unclear that it's just for the two evictions tests, and that it does the same thing as `CACHE_SIZE_IN_BYTES`